edx_api/enrollments/__init__.py

```diff
@@ -36,6 +46,56 @@ def get_student_enrollments(self):
         resp.raise_for_status()
         return Enrollments(resp.json())
 
+    def get_enrollments(self, course_id=None, username=None, usernames=None):
+        """
+        List all course enrollments.
+
+        Args:
+            course_id (str, optional): If used enrollments will be filtered to the specified course id.
+            username (str, optional): Enrollments will be filtered to this specific username.
+            usernames (list, optional): List of usernames to filter enrollments.
+        """
+        params = {'page_size': 1}
```

Comment:
The endpoint here only uses the `course_id` and `username` parameters, so it'll ignore this and any other parameters.
https://github.com/edx/edx-platform/blob/c4b0a6bfcfedd7260fd742b170282105860f68c5/common/djangoapps/enrollment/views.py#L949

Reply:
Yeap, I missed the fact we're using CursorPaginator, I believe if it were a PageNumberPaginator then we could specify the page size and/or page number.